Tests/Editor/PackagesTests.cs

```diff
@@ -0,0 +1,29 @@
+using System.Linq;
+using NUnit.Framework;
+using Unity.ProjectAuditor.Editor;
+
+namespace Unity.ProjectAuditor.EditorTests
+{
+    class PackagesTests : TestFixtureBase
+    {
+        [Test]
+        [TestCase("Project Auditor", "com.unity.project-auditor", "Local", new string[] { "com.unity.nuget.mono-cecil" })]
+        [TestCase("Unity UI", "com.unity.ugui", "BuiltIn", new string[] { "com.unity.modules.ui", "com.unity.modules.imgui" })]
+        [TestCase("Unity Coding Tools", "com.unity.coding", "Registry")]
```

Comment:
I realized _com.unity.coding_ is not always installed. A good replacement would be _com.unity.test-framework_